Source/Core/Core/HW/GCMemcard/GCMemcardDirectory.cpp

```diff
@@ -38,7 +38,7 @@
 #include "Core/HW/Sram.h"
 #include "Core/NetPlayProto.h"
 
-static const char* MC_HDR = "MC_SYSTEM_AREA";
+static constexpr char HEADER_FILENAME[] = "header.bin";
```

Comment:
In case you're wondering: I've intentionally not reused what seems to be an old debugging remnant here, because that could potentially lead to an issue with old builds where the memory card size in the header and the actual memory card size mismatch since the old code didn't check for this. (same for the card text encoding)